ibazel/main.go

```diff
@@ -107,7 +132,7 @@ func main() {
 		switch state {
 		case WAIT:
 			select {
-			case <-sourceFileWatcher.Events:
+			case <-sourceFileEvents:
```

Comment:
Can you refer to the channel as being attached to the object instead of in the function scope?